confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessor.java

```diff
@@ -568,9 +567,13 @@ public TShowConfigNodesResp showConfigNodes() throws TException {
     return configManager.showConfigNodes();
   }
 
-  public void handleClientExit() {}
+  @Override
+  public TShowStorageGroupResp showStorageGroup(List<String> storageGroupPathPattern)
+      throws TException {
+    return configManager.showStorageGroup(new GetStorageGroupPlan(storageGroupPathPattern));
+  }
 
-  // TODO: Interfaces for data operations
+  public void handleClientExit() {}
```

Comment:
What's this empty method used for?

Reply:
This interface was added in March and now it's useless.